src/localization/locales/fr-FR/translation.json

```diff
@@ -0,0 +1,1018 @@
+  "Enter recovery phrase": "Entrez une phrase de recouvrement",
+  "Enter the address in the above field": "Entrez l'adresse dans le champ ci-dessus",
+  "Enter the code generated from your authenticator app.": "Entrez le code généré par votre application d'authentification.",
+  "Enter your PIN and access your device.": "Entrez votre PIN et accédez à votre appareil.",
+  "Enter your PIN.": "Entrez votre PIN.",
+  "Enter {{device}} name": "Entrez le nom de {{device}}",
+  "Error": "Erreur",
+  "Error!": "Erreur !",
+  "Error, malformed request data": "Erreur, données de requête malformées",
+  "Error:": "Erreur :",
+  "Estimated": "Estimé",
+  "Estimated gas units needed to complete the transaction. Includes a small buffer. Not editable for this transaction.":"Unités de gaz nécessaires à la finalisation de la transaction. Petite marge incluse. Non modifiable pour cette transaction.",
+  "Ethereum Address": "Adresse Ethereum",
+  "Euro": "Euro",
+  "Explore Ecosystem": "Explorer l'écosystème",
+  "Explorer": "Explorateur",
+  "Explorer URL": "URL de l'explorateur",
+  "Explorer URL (Optional)": "URL de l'explorateur (optionnelle)",
+  "Export": "Exporter",
+  "Export Cancelled": "Export annulé",
+  "Export Recovery Phrase": "Phrase de recouvrement d'export",
+  "FAQ": "FAQ",
+  "FIDO Device": "Appareil FIDO",
+  "FIDO Device Details": "Détails de l'appareil FIDO",
+  "Failed to connect": "Échec de la connexion",
+  "Failed to load collectibles": "Échec du chargement des articles de collection",
+  "Failed.": "Échec.",
+  "Farming": "Farming",
+  "Fast": "Rapide",
+  "Favorites": "Favoris",
+  "Feature Disabled": "Fonctionnalité désactivée",
+  "Fee": "Frais",
+  "Fee Amount": "Montant des frais",
+  "File Name": "Nom de fichier",
+  "File Upload Failed": "Échec de l'envoi du fichier",
+  "Final": "Final",
+  "Finish Setting Up Recovery Methods": "Mise au point finale des méthodes de recouvrement",
+  "Fireblocks": "Fireblocks",
+  "Fireblocks API credentials can only be configured for accounts imported from Fireblocks": "Les identifiants d'API Fireblocks ne peuvent être configurés que pour les comptes importés depuis Fireblocks",
+  "Fireblocks Approval": "Approbation Fireblocks",
+  "Fireblocks accounts do not support the Testnet Mode currently": "Les comptes Fireblocks ne supportent pas actuellement le mode testnet",
+  "First Request Sent!": "Premire requête envoyée !",
+  "For security reasons your session has timed out. Please log in again.": "Pour votre sécurité, votre temps de session a expiré. Veuillez vous reconnecter.",
+  "For your safety there is a 2 day waiting period to retrieve a phrase.": "Pour votre sécurité, une période de deux jours est nécessaire pour recouvrer une phrase.",
+  "For your security, please create a name and password.": "Pour votre sécurité, veuillez créer un nom et un mot de passe.",
+  "Forgot Password?": "Mot de passe oublié ?",
+  "Forum": "Forum",
+  "French": "Français",
```

Comment:
You should just need to add this as `"French": "French",` to the english translation file

Reply:
Thanks, but are you sure? `yarn scanner` still displays a truckload of errors (same number of lines) even after adding `"French": "French",` to `en/translation.json`. Same, after adding `"French": "Französisch",` to `de-DE/translation.json`. By the way, I was wondering about it, and with this setup a new language should be added in _all_ existing translation files (I can do it for de, es, ru, but zh, hi, ko, ja? no way). Plus it doesn't scale: imagine 30 languages, and adding a 31st. A small redesign of this part seems in order.

Reply:
Yes you are correct, adding new languages will have this issue. In the past we did not have this problem since the number of supported languages was constant. We will re-consider and update the process here. Thank you for the suggestion.
In the meantime, can you please run `yarn scanner` and commit the changed translation files with the new item?

Reply:
All `translation.js` files are now updated, but I still get a humongous amount of error messages when running `yarn scanner`, beginning with:

```
i18next-scanner: Unable to parse Trans component:
var __classPrivateFieldSet = (this && this.__classPrivateFieldSet) || function (receiver, privateMap, value) {
(...8600+ lines...)
```

Reply:
Oh ok. Cloned upstream on my machine (without fr-FR), and `yarn scanner` gives the exact same errors.

Reply:
Thank you! Yes those errors you see are the scanner not being able to parse some of the typescript files. It's not ideal however despite what it looks like, it does it's job.